node/overseer/overseer-gen/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "polkadot-overseer-gen"
+version = "0.1.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2018"
+description = "Generate an overseer including builder pattern and message wrapper from a single struct."
+
+[dependencies]
+tracing = "0.1"
+futures = "0.3"
+async-trait = "0.1"
+thiserror = "1"
+metered = { package = "metered-channel", path = "../../metered-channel" }
+polkadot-overseer-gen-proc-macro = { path = "./proc-macro" }
+polkadot-node-network-protocol = { path = "../../network/protocol"}
+
+# trait SpawnNamed
+sp-core = { git = "https://github.com/paritytech/substrate", branch = "master" }
+futures-timer = "3.0.2"
+tokio-util = { version = "0.6.7", features = [ "time" ] }
```

Comment:
what's tokio-util used for?

Reply:
Hmm, I think that's copy paste. I'll add some `#[deny(unused_dependencies)]`